Workbooks/WorkbooksMetadata.json

```diff
@@ -907,15 +907,15 @@
     "provider": "CyberArk"
   },
   {
-    "workbookKey": "EntityBehaviorAnalyticsWorkbook",
-    "logoFileName": "Microsoft_logo.svg",
-    "description": "Identify compromised users and insider threats using entity behavior analytics. Gain insights into anomalous user behavior from baselines learned from behavior patterns",
+    "workbookKey": "UserEntityBehaviorAnalyticsWorkbook",
+    "logoFileName": "UEBA_logo.svg",
+    "description": "Identify compromised users and insider threats using User and Entity Behavior Analytics. Gain insights into anomalous user behavior from baselines learned from behavior patterns",
     "dataTypesDependencies": [ "BehaviorAnalytics" ],
     "dataConnectorsDependencies": [],
-    "previewImagesFileNames": [ "EntityBehaviorAnalyticsBlack1.png", "EntityBehaviorAnalyticsWhite1.png" ],
+    "previewImagesFileNames": [ "UserEntityBehaviorAnalyticsBlack1.png", "UserEntityBehaviorAnalyticsWhite1.png" ],
     "version": "1.0",
-    "title": "Entity Behavior Analytics",
-    "templateRelativePath": "EntityBehaviorAnalytics.json",
+    "title": "User And Entity Behavior Analytics",
```

Comment:
Please don't change the workbook name if you don't have too because it will affect the customers save functionality